server/src/api/v1/controllers/BookClass.js

```diff
@@ -169,13 +173,15 @@ class BookClass {
       .findById(bookId)
       .then((book) => {
         if (!book) { // check if no book can be found
-          return res.status(404).send({
-            message: 'Sorry, we can\'t find this book'
-          });
+          return res
+            .status(404)
+            .send({
+              message: ['Sorry, we can\'t find this book']
```

Comment:
Curios to know why you're deciding to send the error message as an array.

Reply:
I needed to have a single way of handling error in the client.

Before now, sequelize `errors.errors` sends an array of error messages to the client while other validation checks that are not coming from sequelize just sends a string. 

So I have to check if the error message is an array or a string in the client before I render it. This gives it a polymorphic behaviour.

Please check `client/src/components/flash/FlashMessage` to understand how I render flash messages in the client.